sorcery.lic

```diff
@@ -10,22 +10,35 @@ class Sorcery
   include DRCT
 
   def initialize
+    arg_definitions = [
+      [
+        { name: 'target_mindstate', regex: /\d+/, optional: true, description: 'Repeat the sorcery casting until reaching this mindstate.' }
+      ]
+    ]
+    args = parse_args(arg_definitions)
     @settings = get_settings
 
-    return if cast_sorcery_spell?
+    return if cast_sorcery_spell?(args.target_mindstate)
 
-    focus_runestone
+    focus_runestone(args.target_mindstate)
   end
 
-  def cast_sorcery_spell?
+  def cast_sorcery_spell?(target_mindstate)
     sorcery_spell = @settings.crossing_training_sorcery
     return false unless sorcery_spell
 
     walk_to(@settings.crossing_training_sorcery_room)
 
+    maxcasts = 3
```

Comment:
Probably should do the check once before calling either cast_sorcery_spell or focus_runestone and just return if applicable to avoid running to the sorcery room needlessly. Then we'd just have "if target_mindstate" in each method.